docs/data-cookbooks/hello-world.md

```diff
@@ -12,50 +12,52 @@ Example task meant to be a sanity check/introduction to the Cumulus workflows.
 
-The [workflow definition](workflows/README.md) can be found in [`cumulus/example/workflows/helloworld.yml`](https://github.com/nasa/cumulus/blob/master/example/workflows/helloworld.yml) under `HelloWorldWorkflow:`
-
-```yaml
-HelloWorldWorkflow:
-  Comment: 'Returns Hello World'
-  StartAt: HelloWorld
-  States:
-    HelloWorld:
-      Parameters:
-        cma:
-          event.$: '$'
-          task_config:
-            buckets: '{$.meta.buckets}'
-            provider: '{$.meta.provider}'
-            collection: '{$.meta.collection}'
-      Type: Task
-      Resource: ${HelloWorldLambdaFunction.Arn}
-      Retry:
-        - ErrorEquals:
-          - Lambda.ServiceException
-          - Lambda.AWSLambdaException
-          - Lambda.SdkClientException
-          IntervalSeconds: 2
-          MaxAttempts: 6
-          BackoffRate: 2
-      Next: WorkflowSucceeded
-    WorkflowSucceeded:
-      Type: Succeed
+A workflow definition can be found in the [template repository hello_world_workflow module](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/hello_world_workflow.tf).
+
+```json
+{
+  "Comment": "Returns Hello World",
+  "StartAt": "HelloWorld",
+  "States": {
+    "HelloWorld": {
+      "Parameters": {
+        "cma": {
+          "event.$": "$",
+          "task_config": {
+            "buckets": "{$.meta.buckets}",
+            "provider": "{$.meta.provider}",
+            "collection": "{$.meta.collection}"
+          }
+        }
+      },
+      "Type": "Task",
+      "Resource": "${module.cumulus.hello_world_task_lambda_function_arn}",
+      "Retry": [
+        {
+          "ErrorEquals": [
+            "Lambda.ServiceException",
+            "Lambda.AWSLambdaException",
+            "Lambda.SdkClientException"
+          ],
+          "IntervalSeconds": 2,
+          "MaxAttempts": 6,
+          "BackoffRate": 2
+        }
+      ],
+      "End": true
+    }
+  }
+}
 ```
 
+
 Workflow **error-handling** can be configured as discussed in the [Error-Handling](error-handling.md) cookbook.
 
 ### Task Configuration
 
-The HelloWorld [task itself](workflows/developing-workflow-tasks.md) is defined in [`cumulus/example/lambdas.yml`](https://github.com/nasa/cumulus/blob/master/example/lambdas.yml) under `HelloWorld:`
+The HelloWorld [task](workflows/developing-workflow-tasks.md) is provided for you as part of the `cumulus` terraform module, no configuration is needed.
 
-```yaml
-HelloWorld:
-  handler: index.handler
-  timeout: 300
-  memory: 256
-  source: 'node_modules/@cumulus/hello-world/dist/'
-  useMessageAdapter: true
-```
+If manually adding the resource to a deployment, the lambda resource source is located in the Cumulus source at  [`cumulus/tf-modules/ingest/hello-world-task.tf`](https://github.com/nasa/cumulus/tf-modules/ingest/hello-world-task.tf).  The lambda source code is located in the Cumulus source at ['cumulus/tasks/hello-world'](https://github.com/nasa/cumulus/tasks/hello-world/)
```

Comment:
Could someone actually use the hello world task resource out of the cumulus module?

Reply:
I believe they could add the sub-module directly, same as we add the top module.

Reply:
What sub-module? So their `source` would be `http://github.com/cumulus.zip//tf-modules/ingest`? How could they get only this task?

I'm just not sure what we mean by "manually adding the resource to a deployment". Do you mean the Cumulus module?

Reply:
Sorry, that wasn't stated quite right. 

What I mean is if they wanted the task (say to modify it) for experimentation reasons, they could *totally* grab that file and add it to their (custom module/directory/whatever) with a built lambda and add the lambda to a deployment with updates/whatever.        Perhaps .... we could phrase that a bit differently.   🤔

Reply:
They'd still need *ingest* to run a cumulus workflow for this example.